src/Storefront/Resources/views/storefront/component/product/card/price-unit.html.twig

```diff
@@ -2,7 +2,7 @@
     {% set purchaseUnit = product.purchaseUnit %}
     {% set listingPrice = product.calculatedListingPrice %}
     {% set fromPrice = listingPrice.from %}
-    {% set referncePrice = product.calculatedPrice.referencePrice %}
+    {% set referencePrice = product.calculatedPrice.referencePrice %}
```

Comment:
I think we need a new variable here, with the correct naming and have to deprecate the old one, as it could still be in use